tests/end2end/cypress/integration/feature_toolbar-ghaction.js

```diff
@@ -146,15 +146,231 @@ describe('Feature Toolbar', function () {
+            (req) => {
+                // no cache
+                req.on('before:response', (res) => {
+                    res.headers['cache-control'] = 'no-store'
+                })
+
+                if (req.body.includes('REQUEST=GetFeature')
+                && req.body.includes('TYPENAME=parent_layer')
+                && req.body.includes('dl=1')
+                ) {
+                    // send the modified request and skip any other
+                    // matching request handlers
+                    req.alias = 'GetExport'
+                }
+            }
+        )
+
+        // Click on the export button
+        cy.get('#attribute-layer-main-parent_layer .export-formats > button:nth-child(1)').click({ force: true })
+        cy.get('#attribute-layer-main-parent_layer .export-formats > ul:nth-child(2) > li:nth-child(1) > a:nth-child(1)').click({ force: true })
+
+        cy.wait('@GetExport')
+        .then(({response}) => {
+            cy.fixture('export/export_parent_layer.geojson').then((fixtureGeoJSON) => {
+                expect(response.statusCode).to.eq(200)
+                expect(response.body).to.deep.eq(JSON.parse(fixtureGeoJSON))
+            })
+        })
+
+        // Select the second feature
+        cy.get('#attribute-layer-table-parent_layer lizmap-feature-toolbar[value="parent_layer_d3dc849b_9622_4ad0_8401_ef7d75950111.2"] .feature-select').click({ force: true })
+        cy.wait(300)
+
+        // Click on the export button
+        cy.get('#attribute-layer-main-parent_layer .export-formats > button:nth-child(1)').click({ force: true })
+        cy.get('#attribute-layer-main-parent_layer .export-formats > ul:nth-child(2) > li:nth-child(1) > a:nth-child(1)').click({ force: true })
+
+        cy.wait('@GetExport')
+        .then(({response}) => {
+            cy.fixture('export/export_parent_layer_feature_2.geojson').then((fixtureGeoJSON) => {
+                expect(response.statusCode).to.eq(200)
+                expect(response.body).to.deep.eq(JSON.parse(fixtureGeoJSON))
+            })
+        })
+
+        // Filter the selected feature and export
+        cy.get('#attribute-layer-main-parent_layer .btn-filter-attributeTable').click({ force: true })
+        cy.wait(300)
+
+        // Export
+        cy.get('#attribute-layer-main-parent_layer .export-formats > button:nth-child(1)').click({ force: true })
+        cy.get('#attribute-layer-main-parent_layer .export-formats > ul:nth-child(2) > li:nth-child(1) > a:nth-child(1)').click({ force: true })
+
+        cy.wait('@GetExport')
+        .then(({response}) => {
+            cy.fixture('export/export_parent_layer_feature_2.geojson').then((fixtureGeoJSON) => {
+                expect(response.statusCode).to.eq(200)
+                expect(response.body).to.deep.eq(JSON.parse(fixtureGeoJSON))
+            })
+        })
+
+        // Unfilter
+        cy.get('#attribute-layer-main-parent_layer .btn-filter-attributeTable').click({ force: true })
```

Comment:
Is it needed to unfilter as page get reloaded?

Reply:
nope, will remove it